app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -63,9 +63,18 @@ protected function validator(array $data)
      */
     protected function create(array $data)
     {
+        $user = User::get();
+        // dd(count($user));
+        if (count($user) == 0) {
```

Comment:
You can use ternary operator here to make the code smaller.

Ternary operator looks like this:
`<the-condition> ? <what-to-do-when-condition-is-true> : <what-to-do-when-condition-is-false>;`